modules/govuk/manifests/apps/publishing_api.pp

```diff
@@ -218,4 +218,9 @@
     json    => true,
   }
 
+  @filebeat::prospector { 'publishing_api_sidekiq_json_log':
+    paths  => '/var/apps/publishing-api/log/sidekiq.json.log',
+    fields => {'application' => 'publishing-api-sidekiq'},
```

Comment:
in cases like this where json is hardcoded to true, we can definitely just pass `json => {add_error_key => true}` inline to `filebeat::prospector`